02_configmap_zeebe.yml

```diff
@@ -22,12 +22,7 @@ data:
     
     exec /usr/local/zeebe/bin/broker
   zeebe.cfg.toml: |
-    # Zeebe broker configuration file
```

Comment:
@walt-liuzw the changes on this file are not needed.. can you revert them?

Reply:
@salaboy  hi，i reworked and submitted another PR. so this PR was closed by me.